File: procedural-programming/homework-001/c/exercise-037.c
/* Student: Pedro Henrique Resende Ribeiro
 * Description: Print the price of a discounted product
 * Date: 2024-01-09 - Time: 01:40
 */

#include <stdio.h>

int main() {

    float price, discount, newPrice;

    printf("Enter the price of the product: ");
    scanf("%f", &price);

    discount = 0.12;
    newPrice = price * (1 - discount);

    printf("The new price is %.2f\n", newPrice);

    return 0;

}

File: procedural-programming/homework-001/python/exercise_037.py
# Student: Pedro Henrique Resende Ribeiro
# Description: Print the price of a discounted product
# Date: 2024-01-09 - Time: 01:40

price = float(input('Enter the price of the product: '))

discount = 0.12
newPrice = price * (1 - discount)

print(f'The new price is {newPrice:,.2f}')
